### ISSUE TITLE
Add 'Timing' to Profile

### ISSUE THREAD

**Issue by iambunnieman** [SchoolIdolTomodachi/SchoolIdolAPI#157]
Add 'Timing' to the account section

**Comment by db0company** [comment]
You mean the timing adjustment that you use in the game?

**Comment by iambunnieman** [comment]
Yes. Since they don't show it in game, it would be nice to have it on schoolido.lu for when I transfer between devices.

**Comment by db0company** [comment]
I see. This might be interesting, indeed. Thank you for your suggestion.
